chrome/locale/de/dkim.properties

```diff
@@ -102,3 +102,4 @@ DKIM_SIGWARNING_FROM_NOT_IN_SDID	= Absender gehört nicht zur Domain
 DKIM_SIGWARNING_FROM_NOT_IN_AUID	= Absender passt nicht zur Benutzeridentität
 
 # DKIM_SIGWARNING - POLICY
+DKIM_SIGWARNING_UNSIGNED_HEADER = Kopfzeile '%S' ist nicht signiert
```

Comment:
Files should end with a newline (which in many editors then shows up as a single empty line at the end).

Reply:
Fixed